model/task/db.go

```diff
@@ -2833,7 +2837,6 @@ func abortAndMarkResetTasks(ctx context.Context, filter bson.M, taskIDs []string
 					AbortedKey:           true,
 					AbortInfoKey:         AbortInfo{User: caller},
 					ResetWhenFinishedKey: true,
-					DisplayStatusKey:     DisplayStatusExpression,
```

Comment:
This was mistakenly added in #8519 😓 I will need to run a DB operation to unset the field:
```shell
db.tasks.updateMany({}, { $unset: { display_status: 1 } })
```